lib/shelley/bench/Latency.hs

```diff
@@ -151,11 +152,8 @@ walletApiBench capture ctx = do
     fmtTitle "Latencies for 10 fixture wallets scenario"
     runScenario (nFixtureWallet 10)
 
-    {-- PENDING: We currently have a limited amount of available fixture
-       wallets, so we can't just run a benchmark with 100 wallets in parallel.
-    fmtTitle "Latencies for 100 fixture wallets scenario"
+    fmtTitle "Latencies for 100 fixture wallets"
     runScenario (nFixtureWallet 100)
```

Comment:
The benchmark took 8 minutes for me to run. When we create 100 wallets we repeatedly run `postWallet >> waitForFunds`, instead of first running all `postWallet` and then waiting for all wallets to have balance concurrently.

Changing to the latter approach would probably speed it up a bit, but I'm not sure how important it is that we do.